rules/web/web_cve_2022_42889_text4shell_exploit.yml

```diff
@@ -0,0 +1,27 @@
+title: Text4Shell Exploit CVE-2022-42889
```

Comment:
It is not realy the Exploit but a Payload that is detected.
For the exploit , we need the server log and find java run a curl.